packages/data-service/src/instance-detail-helper.js

```diff
@@ -35,13 +35,13 @@ function getReadPreferenceOptions(db) {
 function getStats(results, done) {
   const databases = results.databases;
 
-  const keys = ['document_count', 'storage_size', 'index_count', 'index_size'];
+  const keys = ["document_count", "storage_size", "index_count", "index_size"];
   const stats = {};
-  keys.map(function(k) {
+  keys.map(function (k) {
     stats[k] = 0;
   });
-  databases.map(function(db) {
-    keys.map(function(k) {
+  databases.map(function (db) {
```

Comment:
I think usually we don't have a space between function and brackets. I think your prettifier messed up some code here :)